sdk/src/config.rs

```diff
@@ -251,6 +255,8 @@ pub enum GadgetCLICoreSettings {
         blueprint_id: u64,
         #[structopt(long)]
         service_id: Option<u64>,
+        #[structopt(long, parse(try_from_str))]
+        protocol: Protocol,
```

Comment:
Could just have it default to Tangle.

Reply:
I want these mandatory to not have silent bugs in the future that might cause the failing of passing the protocol argument.